### PR TITLE
feat(controller): support plugin assets serving

### PR DESCRIPTION
**Pull request by jialeicui**
## Description
Support plugin assets serving

cc @waynelwz 

## Modules
- [ ] UI
- [x] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [x] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: server/controller/src/main/java/ai/starwhale/mlops/domain/panel/PluginService.java [hunk 1]
```diff
@@ -121,4 +140,34 @@ public void uninstallPlugin(String idStr) {
     public PageInfo<PanelPluginVo> listPlugin() {
         return PageUtil.toPageInfo(panelPluginMapper.list(), panelPluginConvertor::convert);
     }
+
+    public void initPluginCache(InputStream is, String subPath) throws IOException, ArchiveException {
```
Comment (anda-ren) [inline server/controller/src/main/java/ai/starwhale/mlops/domain/panel/PluginService.java:144]: private method?

File: server/controller/src/main/java/ai/starwhale/mlops/domain/panel/PluginService.java [hunk 2]
```diff
@@ -121,4 +140,34 @@ public void uninstallPlugin(String idStr) {
...
+        if (cachePath.isEmpty()) {
+            log.warn("cache path is empty");
+            return;
+        }
+        TarFileUtil.extract(is, Paths.get(cachePath, subPath).toString());
+    }
+
+    public void initPluginCaches() {
```
Comment (anda-ren) [inline server/controller/src/main/java/ai/starwhale/mlops/domain/panel/PluginService.java:152]: private method?

File: server/controller/src/main/java/ai/starwhale/mlops/domain/panel/PluginService.java (hunk 1 above)
Comment (jialeicui) [reply server/controller/src/main/java/ai/starwhale/mlops/domain/panel/PluginService.java]: Fixed

File: server/controller/src/main/java/ai/starwhale/mlops/domain/panel/PluginService.java (hunk 2 above)
Comment (jialeicui) [reply server/controller/src/main/java/ai/starwhale/mlops/domain/panel/PluginService.java]: Fixed

**Review by anda-ren — DISMISSED** [review]
lgtm

**Review by anda-ren — APPROVED** [review]

**Pull request merged**